ontology_api/functions/functions.py

```diff
@@ -0,0 +1,54 @@
+from owlready2 import *
+
+
+models = dict()
```

Comment:
Each ontology loaded with create_model() needs to have its own unique ID. We store it in a dictionary that uses the model IDs as keys and the models as values. In case a model is somehow deleted from the structure, the other models still keep their IDs (implementing that with a list would be more difficult, since the indices would change).